### PR TITLE
multiple pages: add korean translation

### PR DESCRIPTION
**Pull request by lch7167**
translation to korean

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by einverne** [comment]
hi @lch7167 You do note need to close pull request, just make some changes and push.

**Comment by lch7167** [comment]
hi @einverne i already signed the CLA, but "CLA not signed yet" is still pending. is it no problem?

**Comment by einverne** [comment]
Do you use the same email address in GitHub?

**Comment by lch7167** [comment]
@einverne first commit "lch" is my github desktop. it used another email, so i changed the name lch7167 and i wanted to change the email to youngsj69@gmail.com same as this account. but i mistake changed the email to youngsj69@gamil.com.

**Comment by einverne** [comment]
CLA check now passed!

File: pages.ko/autossh.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# autossh
+
+> SSH 연결을 실행, 모니터링 및 재시작. port 재전송 tunnel을 유지하기 위해 자동 재연결. 모든 ssh 플래그 허용. 더 많은 정보는: https://harding.motd.ca/autossh.
```
Comment (einverne) [inline pages.ko/autossh.md]:
```suggestion
> SSH 연결을 실행, 모니터링 및 재시작. port 재전송 tunnel을 유지하기 위해 자동 재연결. 모든 ssh 플래그 허용. 
> 더 많은 정보는: <https://harding.motd.ca/autossh>.
```
We usually use this format.

File: pages.ko/avrdude.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# avrdude
+
+> Atmel AVR 마이크로 컨트롤러 프로그래밍을 위한 드라이버 프로그램. 더많은 정보는: https://www.nongnu.org/avrdude/.
```
Comment (einverne) [inline pages.ko/avrdude.md]: The same.

File: pages.ko/awk.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# awk
+
+> 파일 작업을 위한 다목적 프로그래밍 언어. 더 많은 정보는: https://github.com/onetrueawk/awk.
```
Comment (einverne) [inline pages.ko/awk.md]: Same here.

**Comment by lch7167** [comment]
@einverne thank you :)

**Comment by lch7167** [comment]
@einverne i want to put multiple pages in a common folder and add other multiple pages. what should i do??

**Comment by einverne** [comment]
You want to add more new commands? Or you want to translate more? You can take a look at [CONTRIBUTING](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) page.

**Comment by sbrl** [comment]
You're welcome to open a new PR @lch7167 :-)

**Pull request merged**